apps/site/assets/ts/schedule/__tests__/ScheduleLoaderTest.tsx

```diff
@@ -894,4 +896,19 @@ describe("ScheduleLoader", () => {
 
     wrapper.unmount();
   });
```

Comment:
Let's add a test to verify that `<ScheduleLoader />` shows the `ScheduleDirection` component in this situation.